BuildTools/.swiftformat

```diff
@@ -1,8 +1,8 @@
 --indent 4
 --patternlet inline
---disable unusedArguments
+--disable unusedArguments,hoistAwait,hoistTry
```

Comment:
Why did we decide to disable these?

Reply:
They are not quite there yet:

<img width="256" alt="image" src="https://github.com/WeTransfer/WeTransfer-iOS-CI/assets/5387088/d1db6876-f201-47b4-8b9c-000dbfb529be">